bin/callback.js

```diff
@@ -0,0 +1,84 @@
+const http = require('http')
+
+const CALLBACK_URL = process.env.CALLBACK_URL ? new URL(process.env.CALLBACK_URL) : null
+const CALLBACK_TIMEOUT = process.env.CALLBACK_TIMEOUT || 5000
+const CALLBACK_OBJECTS = process.env.CALLBACK_OBJECTS ? JSON.parse(process.env.CALLBACK_OBJECTS) : {}
+
+/**
+ * @return {Boolean}
+ */
+exports.isCallbackSet = () => {
```

Comment:
Can you please make this a boolean export (not a function call).

Reply:
Changed